File: Assignment_2/Assignment2_3.py
"""-------------------------------------------------------------------------------------------
                          Assignment2_3
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
Problem statement : This program accepets a number from user and displays
                    factorial of given number
------------------------------------------------------------------------------------------"""
from Arithmatic import Mult
#----------------------------------------------------------------------------------------
# This function calculates factorial of given number
#---------------------------------------------------------------------------------------

def calculate_factorial(number):
    result = 1
    for num in range(2,number+1):
        result = Mult(result,num)
    return result
#------------------------------------------------------------------------------------------
# This function accepts a input number from user and calls factorial function
#------------------------------------------------------------------------------------------

def main():
    number=int(input("Enter number to calculate factorial:"))
    result=calculate_factorial(number)
    print(f"Factorial of {number} is :",result)
    
#-----------------------------------------------------------------------------------
# This is main entry point of the program
#-----------------------------------------------------------------------------------
if __name__=="__main__":
    main()
#-----------------------------------------------------------------------------------

File: JavaPractice/Assignment_2/Assignment2_3.java

/*-------------------------------------------------------------------------------------------
                          Assignment2_3
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
Problem statement : This program accepets a number from user and displays
                    factorial of given number
------------------------------------------------------------------------------------------*/
import java.util.Scanner;




public class Assignment2_3 {
	public static void calculate_Factorial(int number) {
		int fact=1;
		for(int cnt=1;cnt<=number;cnt++) {
			fact=AssignmentModule.Multiply(fact,cnt);
			
		}
		System.out.println("Factorial of "+number+" Is:"+fact);
		
		
	}

	public static void main(String[] args) {
		Scanner inputObj=new Scanner(System.in);
		System.out.println("Enter the number:");
		int number=inputObj.nextInt();
		calculate_Factorial(number);
		inputObj.close();
		

	}

}
